src/app/feedback/[id]/page.tsx

```diff
@@ -0,0 +1,52 @@
+"use client";
+
+import Sidebar from "@/components/sidebar/sidebar";
+import { getBookingInfo, getBoatInfo } from "@/services/notion.service";
+import { useEffect, useState } from "react";
+import router from "next/router";
+import FeedbackForm from "@/components/feedback";
+
+export default function UserFeedbackPage({ params }: { params: { id: string } }) {
+  const [boatInfo, setBoatInfo] = useState<any>({});
```

Comment:
`any` -> Boat | null.

By default objects shall not be `{}` cos otherwise is hard to check are defined or not.